ADAM/src/AppBundle/Resources/views/advert/show.html.twig

```diff
@@ -70,3 +62,17 @@
         </div>
 </section>
 {% endblock %}
+
+{% block javascripts %}
+    <script type="text/javascript">
+        $('#myModal').modal({
+            backdrop: "static",
+            keyboard: false
+        }); 
+        
+        $(".supress").click(function(){
+            var actionadvert = $(this).attr('data-href');
```

Comment:
Utilise plutôt `var actionadvert = $(this).data('href');` qui a le même comportement et qui est prévu pour ce genre d'utilisation